sapi/fpm/fpm/fpm_php.c

```diff
@@ -93,7 +93,21 @@ int fpm_php_apply_defines_ex(struct key_value_s *kv, int mode) /* {{{ */
 	if (!strcmp(name, "extension") && *value) {
 		zval zv;
 		zend_interned_strings_switch_storage(0);
+
+#if ZEND_RC_DEBUG
+		bool orig_rc_debug = zend_rc_debug;
+		/* FIXME: Loading extensions after php_module_startup() breaks some
+		 * invariants. For instance, it will update the refcount of persistent
+		 * strings, which is normally not allowed at this stage. */
```

Comment:
I'm not sure if there is much point in adding this as it's not really easy to fix. It is actually best described here: https://github.com/php/php-src/pull/12233#issuecomment-1732052966 . It might make sense to create an issue for tracking this problem. I can't however thing about another solution than the pool manager: https://github.com/php/php-src/issues/11723 so we might as well update just that issue unless you see a different way to address this.

Reply:
It's not related to interned strings, it's a different issue. I will remove the FIXME tag, but I think the comment is useful to remember why we are doing this.

Reply:
Yeah I know. What I meant is that we cannot easily fix loading extension after the module startup. Comment why it's done is useful.

Reply:
I agree. I've removed the "FIXME" tag